project/src/main/java/com/google/sps/servlets/VolunteeringFormHandlerServlet.java

```diff
@@ -78,4 +81,8 @@ private static void updateVolunteeringOpportunityInDB(
     // TO DO: change eventId to parameter value
     SpannerTasks.updateVolunteeringOpportunity(opportunity);
   }
+
+  private static List<String> splitAsList(String values) {
+    return Arrays.asList(values.split("\\s*,\\s*"));
+  }
```

Comment:
Why you need to return a List? Maybe return a HashSet directly?

Reply:
Done - agreed.